router/server.go

```diff
@@ -214,6 +238,12 @@ func (s *Server) ServeHTTP(isSecure bool, w http.ResponseWriter, req *http.Reque
 		return
 	}
 
+	if req.RequestURI == "/bucky/v1/send" {
+		s.resendBucky(req)
```

Comment:
I think the `fmt.Fprintfw(w, "")` should be written inside `resendBucky`, so that the order of things, and the fact that no delay is introduced, is clear.